api/src/main/java/org/openmrs/module/adminui/account/Account.java

```diff
@@ -130,7 +131,27 @@ public void setGender(String gender) {
 	public String getGender() {
 		return getPerson().getGender();
 	}
-	
+
+	public List<PersonAttribute> getActiveAttributes() {
+		return getPerson().getActiveAttributes();
+	}
+
+	/**
+	 * Gets the person attribute for the specified person for the getPersonAttributeTypeByUuid
```

Comment:
Does the javadoc have to talk about implementation details? (The actual API method being called)

Reply:
Removed